### PR TITLE
Disable history-import command

### PR DESCRIPTION
**Pull request by qfel**
Temporarily disable the `history-import` command

See #14076 for details.

### CODE REVIEW

**Comment by NotTheDr01ds** [comment]
Up to @fdncred  but I think we're probably too close (2 days) to next release to have a command in the code that simply errors.

It *would* be more convenient for testing, though, so perhaps it could go back in after the release looks good?

**Comment by fdncred** [comment]
I don't really want to land a disabled command. Let's just wait for the investigation and potential fix.

**Comment by NotTheDr01ds** [comment]
Right - I can always just checkout the commit with the fix to test.

File: crates/nu-cli/src/commands/history/history_import.rs [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+        };
+        let Some(current_history_path) = history.file_path() else {
+            return Err(ShellError::ConfigDirNotFound {
+                span: Some(call.head),
+            });
+        };
+
+        match input {
+            PipelineData::Empty => {
+                let other_format = match history.file_format {
```
Comment (NotTheDr01ds) [inline crates/nu-cli/src/commands/history/history_import.rs:90]: If the user has *switched* their `$env.config.history.file_format`, then they are going to have *both* types of history files present.  If I'm reading this correctly, then this `match` means that `history import` is always going to import text-backed to SQLite in that case.  While that's the most likely scenario, it doesn't make allowance for someone who is switching back to text-backed for some reason.  Wouldn't it be better to `match` on `$env.config.history.file_format` state?

**Pull request closed without merging**